README.md

```diff
@@ -15,122 +15,228 @@ npm install convert-units --save
+convert(12000).from('mm').toBest({ exclude: ['m'] });
 // { val: 1200, unit: 'cm', plural: 'Centimeters' } (the smallest unit excluding meters)
 
 convert(900).from('mm').toBest({ cutOffNumber: 10 });
 // { val: 90, unit: 'cm', plural: 'Centimeters' } (the smallest unit with a value equal to or above 10)
 
-convert(1000).from('mm').toBest({ cutOffNumber: 10 })
+convert(1000).from('mm').toBest({ cutOffNumber: 10 });
 // { val: 100, unit: 'cm', plural: 'Centimeters' } (the smallest unit with a value equal to or above 10)
 ```
 
-You can get a list of the measurement types supported with `.measures`
+You can get a list of the measures available to the current instance with `.measures`
 
 ```js
-convert().measures()
+convert().measures();
 // [ 'length', 'mass', 'volume' ]
+
+const differentConvert = configureMeasurements({
+    volume,
+    mass,
+    length,
+    area,
+});
+differentConvert().measures();
+// [ 'length', 'mass', 'volume', 'area' ]
 ```
 
 If you ever want to know the possible conversions for a unit, just use `.possibilities`
 
 ```js
-convert().from('l').possibilities()
+convert().from('l').possibilities();
 // [ 'ml', 'l', 'tsp', 'Tbs', 'fl-oz', 'cup', 'pnt', 'qt', 'gal' ]
 
-convert().from('kg').possibilities()
+convert().from('kg').possibilities();
 // [ 'mcg', 'mg', 'g', 'kg', 'oz', 'lb' ]
 ```
 
 You can also get the possible conversions for a measure:
 ```js
-convert().possibilities('mass')
+convert().possibilities('mass');
 // [ 'mcg', 'mg', 'g', 'kg', 'oz', 'lb', 'mt', 't' ]
 ```
 
 You can also get the all the available units:
 ```js
-convert().possibilities()
+convert().possibilities();
 // [ 'mm', 'cm', 'm', 'in', 'ft-us', 'ft', 'mi', 'mcg', 'mg', 'g', 'kg', 'oz', 'lb', 'mt', 't', 'ml', 'l', 'tsp', 'Tbs', 'fl-oz', 'cup', 'pnt', 'qt', 'gal', 'ea', 'dz' ];
 ```
 
 To get a detailed description of a unit, use `describe`
 
 ```js
-convert().describe('kg')
+convert().describe('kg');
 /*
   {
-    abbr: 'kg'
-  , measure: 'mass'
-  , system: 'metric'
-  , singular: 'Kilogram'
-  , plural: 'Kilograms'
+    abbr: 'kg',
+    measure: 'mass',
+    system: 'metric',
+    singular: 'Kilogram',
+    plural: 'Kilograms',
   }
 */
 ```
 
 To get detailed descriptions of all units, use `list`.
 
 ```js
-convert().list()
+convert().list();
 /*
   [{
-    abbr: 'kg'
-  , measure: 'mass'
-  , system: 'metric'
-  , singular: 'Kilogram'
-  , plural: 'Kilograms'
+    abbr: 'kg',
+    measure: 'mass',
+    system: 'metric',
+    singular: 'Kilogram',
+    plural: 'Kilograms',
   }, ...]
 */
 ```
 
 You can also get detailed descriptions of all units for a measure:
 
 ```js
-convert().list('mass')
+convert().list('mass');
 /*
   [{
-    abbr: 'kg'
-  , measure: 'mass'
-  , system: 'metric'
-  , singular: 'Kilogram'
-  , plural: 'Kilograms'
+    abbr: 'kg',
+    measure: 'mass',
+    system: 'metric',
+    singular: 'Kilogram',
+    plural: 'Kilograms',
   }, ...]
 */
 ```
 
-Supported Units
+Custom Measures
```

Comment:
Do you think the section "Want More?" can be removed from the readme now you have this section?

Reply:
How about renaming the section to something like, "Request Packaged Measures"? I still want folks to send in PRs with measures they would like included (packaged with the lib). Saves everyone the effort of creating the measures themselves. While still allowing for custom measures and even the ability to modify/override packaged measures.